### ISSUE TITLE
Errors when scraping validation set

### ISSUE THREAD

**Issue by fsingletonthorn** [fsingletonthorn/EffectSizeScraping#59]
The following bugs out:
```{r}
scrapePMC(call = "https://www.ncbi.nlm.nih.gov/pmc/oai/oai.cgi?verb=GetRecord&identifier=oai:pubmedcentral.nih.gov:1215486&metadataPrefix=pmc",  ftp = "ftp://ftp.ncbi.nlm.nih.gov/pub/pmc/oa_package/7d/65/PMC1215486.tar.gz", statcheck = T)
```
`Error in stringr::str_split(cleanedTestStat, "(?<!n|N|df|DF|df2)(=|:)",  : 
  subscript out of bounds`

**Comment by fsingletonthorn** [comment]
Error in the extraction function means that non-F tests can in rare circumstances be extracted which breaks things up the line

**Comment by fsingletonthorn** [comment]
Narrowed down to F statistic picking up issues with: 
```{r}
extractTestStats("sion of PlxnB3 in cultivated primary cerebellar neurons isolated from six days old c57BL/6J mice (a-f; 400 × magnification) but not in astrocytes (g-i) or oligodendrocytes (j-1) (630 × magnification). The c")
```
```

**Comment by fsingletonthorn** [comment]
Fixed by 35ad0f5f4786e5fa6fde3faa0269e82ee390b3ce
